src/timeseries.jl

```diff
@@ -0,0 +1,166 @@
+export veltimeseries
+
+"""
+    veltimeseries(notes::MIDI.Notes, pitch::UInt8, grid = 0:1//notes.tpq:1, zeros::Bool = false, av::Bool = false)
+
+Generate a time series of velocities out of given notes.
+Arguments:\n
+- `notes`:  the notes from which the time series is generated
+- `pitch`:  specify the pitches that shall be considered for the time series
+- `grid`:   give a grid to which the notes will be quantized before time
+            series generation. See [`quantize`](@ref) for details.
+- `zeros`:  put zeros on gridpoints/ticks where no note occurs
+- `av`:     If multiple notes appear at one gridpoint, the highest velocity is
+            chosen by default. Set this to true to average over velocities at
+            that grid point
+"""
+function veltimeseries(notes::MIDI.Notes, pitch::Vector{UInt8}, grid = 0:1//notes.tpq:1, zeros::Bool = false, av::Bool = false)
+    # get only the needed pitches
+    pnotes = purgepitches(notes, pitch)
```

Comment:
Is it not possible to have a version that does not `purgepitches`, but instead gets the average velocity out of all notes in a given bin, irrespectively of their pitch?? (it definitely is possible)

This we also need anyway, so you should do it in this PR as well.

Reply:
In addition, the argument `average / maximum` can also propagate to this method, i.e. take the average or maximum depending on the Boolean.